crypto/dh/dh_key.c

```diff
@@ -325,6 +325,9 @@ static int generate_key(DH *dh)
 #ifdef FIPS_MODULE
             if (dh->params.q == NULL)
                 goto err;
+# ifndef OPENSSL_NO_FIPS186_4_FFC
```

Comment:
Condition out the above `if` statement as well?

Reply:
if true, it goes to err; if false scroll scroll scroll it also goes to err.

yeah, agree.